transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -43,6 +38,12 @@
 import io.netty.util.internal.RecyclableArrayList;
 import io.netty.util.internal.logging.InternalLogger;
 import io.netty.util.internal.logging.InternalLoggerFactory;
+import javafx.util.Pair;
```

Comment:
please not use `Pair`... Just use `AbstractMap.SimpleEntry` or create your own class to hold this pair.

Reply:
Fixed.
Use AbstractMap.SimpleEntry to avoid introducing more code.